spec/models/factory_spec.rb

```diff
@@ -144,4 +144,24 @@
       expect(factory1.depth).to eq 3
     end
   end
+
+  describe ".family" do
+    let!(:factory1) { create :factory }
+    let!(:factory2) { create :factory }
+    let!(:factory3) { create :factory }
+    let!(:factory4) { create :factory }
+    let!(:asso1) { create :asso, name: "asso1", factory: factory2 }
+    let!(:asso_relation1) { AssoRelation.create(factory_id: factory1.id, asso_id: asso1.id) }
+    let!(:asso2) { create :asso, name: "asso2", factory: factory3 }
+    let!(:asso_relation2) { AssoRelation.create(factory_id: factory1.id, asso_id: asso2.id) }
+    let!(:asso3) { create :asso, name: "asso3", factory: factory4 }
+    let!(:asso_relation3) { AssoRelation.create(factory_id: factory3.id, asso_id: asso3.id) }
+
+    it do
+      expect(factory1.family).to eq [
+                                      [factory2, factory3],
```

Comment:
Style/IndentArray: Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.